docs/walkthrough/walkthrough_data.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Walkthrough data
+summary: Example data for walkthrough and testing
+permalink: walkthrough_data.html
+datatable: true
+---
+{% assign walkthrough_folder = "walkthrough/data/" %}
+
+This page describes the dataset that is used in the 
+[Walkthrough](walkthrough.html) for testing _ReadTools_ binaries.
+A list with all the files and checksum could be found 
```

Comment:
It is an unique file with a list of the files with corresponding checksum. Maybe: "A checksum for available files can be found here"? I changed it for that!